packages/framework/bundle/heatmap/bundle.js

```diff
@@ -41,11 +41,11 @@ Oskari.clazz.define("Oskari.mapframework.bundle.heatmap.HeatmapBundle", function
             "src" : "../../../../bundles/mapping/heatmap/resources/css/style.css"
         }, {
         // NOTE! EXTERNAL LIBRARY!
-            "type": "text/javascript",
-            "src": "../../../../libraries/colpick-jQuery-Color-Picker-master/js/colpick.js"
+            "type" : "text/javascript",
+            "src" : "../../../../libraries/spectrum/spectrum-min.js"
```

Comment:
ColorPickerInput.js depends on spectrum, so it should be divmanazer's dependency, not heatmap's